laborator/content/buffer-overflow/solution/9-return-address/Readme.md

```diff
@@ -0,0 +1,14 @@
+# Writeup
+
+- First use this command to scan the exec:
+> ```Bash
+> objdump -M intel -d break_this
+> ```
+- `magic_function()` is saved at `0x08048596`
+- From the first 4 lines from `read_buffer()` we get that: `ebp` and `edi` get pushed on stack then the stack is extended by `0x54` = `84`
```

Comment:
```suggestion
- From the first 4 lines from `read_buffer()` we get that: `ebp` and `edi` get pushed on the stack then the stack is extended by `0x54` = `84`
```
Also include a snippet of those 4 lines so that the explanation is clearer.